### PR TITLE
Email validation

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `Email validation`
**In this milestone**
I and my partner added the following:
JavaScript(to do validation of email input)

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const email = document.getElementById('#email');
+const form = document.querySelector('#mobile-form');
+
+const errorMessage = document.querySelector('.error-message');
+
+const validateEmail = (e) => {
```
Comment (SajjadAhmad14) [inline form.js]: Please implement validation for email so if a user submits the contact form with the email field containing uppercase characters an error message should show up. Moreover, the error message should show up near the submit button and should follow the design from the original design page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Gabriel Nwachukwu, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: form.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function validation() {
```
Comment (SajjadAhmad14) [inline form.js]: The error should be descriptive when the user fills the email with the upper case. The current error message is `*Make sure to fill your name` even all the fields are filled. Kindly see the screenshot below and fix the issue.
upper

File: form.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const form = document.getElementById('form');
```
Comment (SajjadAhmad14) [inline form.js]: Users should be allowed to send messages and forms should be submitted when the email field is in lower case. Kindly see the screenshot below and fix the issue.
lower

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Gabriel Nwachukwu, 
 ### Thank you for working on the changes suggested previously.  :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: form.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  const name = form.elements.name.value;
+  const email = form.elements.email.value;
+  const comment = form.elements.comment.value;
+  const message = document.getElementById('message');
+  message.innerHTML = '';
+
+  form.addEventListener('submit', (event) => {
```
Comment (rloterh) [inline form.js]: - [ ] OPTIONAL
The form is accepting `numbers` as inputs for both _name_ and _message_ columns. Kindly add some validations to check on such entries

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Gabriel, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**